acq4/devices/PMT/PMT.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from acq4.devices.Device import *
+from acq4.devices.OptomechDevice import OptomechDevice
+from acq4.devices.DAQGeneric import DAQGeneric
+
+class PMT(DAQGeneric, OptomechDevice):
```

Comment:
I didn't want to rename PMT everywhere else to PhotoDetector (e.g. for example as detector in the Imager module). Should I name the class PhotoDetector and rename PMT everywhere to PhotoDetector or is there another way?